web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -39,10 +39,16 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
 
         this._disableDateTimeFields();
         this._attachTemporalEvents();
+
+        if (this.layer.temporalextentloaded) {
```

Comment:
Would it be better/possible to store this ui state locally rather than on the layer